src/filter-functions/category.js

```diff
@@ -0,0 +1,20 @@
+export const categoryHandler=(data,state)=>{
+    let arr=[]
```

Comment:
try to be more specific with naming of objects and arrays.
For example - you can write it as `categoryList`

Reply:
right will do it